SQL/0000-00-00-schema.sql

```diff
@@ -2014,3 +2018,40 @@ CREATE TABLE `feedback_mri_comments` (
   CONSTRAINT `FK_feedback_mri_comments_2` FOREIGN KEY (`PredefinedCommentID`) REFERENCES `feedback_mri_predefined_comments` (`PredefinedCommentID`),
   CONSTRAINT `FK_feedback_mri_comments_3` FOREIGN KEY (`FileID`) REFERENCES `files` (`FileID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- ********************************
+-- Consent tables
+-- ********************************
+
+CREATE TABLE `consent` (
+  `ConsentID` int(2) NOT NULL AUTO_INCREMENT,
```

Comment:
auto_increment should be unsigned int.

also. I might have pushed a little bit to much to reduce as much as possible the size of attributes.
I during setup and test, someone insert then delete some 80 consents, then not much numbers are left unless the auto_increment is reseted prior going live. Maybe 2 bytes would be preferable. Plus int(2) is still equivalent to long int(2) which take 8 bytes

Reply:
@PapillonMcGill sorry I don't know if I'm understanding you correctly.. are you saying we should stick with the int(2) for ConsentID?

Reply:
No, the opposite, more like `unsigned small int(5)` (2 bytes) but generally it is ok to leave the auto_incremented ID to `unsigned int()` or `unsigned int(11)` so it as the maximum range possible, well not really but high enough to (almost) never reach it.

We are not constrained as in the early `80 with 8k of ram in a system.